<!DOCTYPE html>
<html lang="en" prefix="og: http://ogp.me/ns# fb: https://www.facebook.com/2008/fbml">
<head>
    <title>Creating a functional gene database, obstacles and a quick tree _anvio_ - Exploring Microbes</title>
    <!-- Using the latest rendering mode for IE -->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">


    <link href="/images/favicon.png" rel="icon">

<link rel="canonical" href="/blog/func_genes_db_anvio/">

        <meta name="author" content="Carl-Eric" />
        <meta name="keywords" content="bioinformatics,tutorial" />
        <meta name="description" content="Assembling a quick MxaF/XoxF sequence database and using anvio for a first shot at treeing." />

        <meta property="og:site_name" content="Exploring Microbes" />
        <meta property="og:type" content="article"/>
        <meta property="og:title" content="Creating a functional gene database, obstacles and a quick tree _anvio_"/>
        <meta property="og:url" content="/blog/func_genes_db_anvio/"/>
        <meta property="og:description" content="Assembling a quick MxaF/XoxF sequence database and using anvio for a first shot at treeing."/>
        <meta property="article:published_time" content="2019-01-25" />
            <meta property="article:section" content="Blog." />
            <meta property="article:tag" content="bioinformatics" />
            <meta property="article:tag" content="tutorial" />
            <meta property="article:author" content="Carl-Eric" />



    <!-- Bootstrap -->
        <link rel="stylesheet" href="/theme/css/bootstrap.sandstone.min.css" type="text/css"/>
    <link href="/theme/css/font-awesome.min.css" rel="stylesheet">
    <link href="/theme/css/pygments/native.css" rel="stylesheet">
    <link rel="stylesheet" href="/theme/css/style.css" type="text/css"/>
        <link href="/theme/css/custom.css" rel="stylesheet">





</head>
<body>

<div class="navbar navbar-default navbar-fixed-top" role="navigation">
	<div class="container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a href="/" class="navbar-brand">
<img class="img-responsive pull-left gap-right" src="/images/logo.png" width="125"/>             </a>
        </div>
        <div class="collapse navbar-collapse navbar-ex1-collapse">
            <ul class="nav navbar-nav">
                         <li><a href="/pages/about-me.html">
                             About me.
                          </a></li>
                         <li><a href="/pages/publications.html">
                             Publications.
                          </a></li>
                         <li><a href="/pages/research.html">
                             Research.
                          </a></li>
                         <li><a href="/pages/teaching.html">
                             Teaching.
                          </a></li>
                        <li class="active">
                            <a href="/category/blog.html">Blog.</a>
                        </li>
            </ul>
            <ul class="nav navbar-nav navbar-right">
            </ul>
        </div>
        <!-- /.navbar-collapse -->
    </div>
</div> <!-- /.navbar -->
<!-- Banner -->
<!-- End Banner -->
<div class="container">
    <div class="row">
        <div class="col-sm-9">
    <section id="content">
        <article>
            <header class="page-header">
                <h1>
                    <a href="/blog/func_genes_db_anvio/"
                       rel="bookmark"
                       title="Permalink to Creating a functional gene database, obstacles and a quick tree _anvio_">
                        Creating a functional gene database, obstacles and a quick tree _anvio_
                    </a>
                </h1>
            </header>
            <div class="entry-content">
                <div class="panel">
                    <div class="panel-body">
<footer class="post-info">
    <span class="label label-default">Date</span>
    <span class="published">
        <i class="fa fa-calendar"></i><time datetime="2019-01-25T00:00:00+01:00"> Fri 25 January 2019</time>
    </span>





<span class="label label-default">Tags</span>
	<a href="/tags/bioinformatics.html">bioinformatics</a>
        /
	<a href="/tags/tutorial.html">tutorial</a>
    
</footer><!-- /.post-info -->                    </div>
                </div>
                <style>
body {
text-align: justify}
</style>

<h2>What is this about?</h2>
<p>From work related to the soft coal slags that I started to study a <a href="https://www.exploringmicrobes.de/pages/research">little while ago</a> I obtained a couple of isolates that turned out to be rather worthwhile for some follow up work. Based on a functional gene-targeting PCR I could see early on that these isolates carry <em>xoxF</em> genes. So far so good, <em>xoxF</em> encodes one type of methanol dehydrogenases (MDH) (yes the isolates are methylotrophs). For a long time it was commonly accepted that menthanol oxidation in methylotrophs is mediated by MxaFI-type MDHs. This perception changed recently, and meanwhile it is hypothesized that XoxF-type MDHs are more widespread in the environment and ecologically more relevant. </p>
<p>If you are interested in methylotrophy and methanol dehydrogenases (especially MxaFI- versus XoxF-type MDHs) I recommend to have a look at these two publications:</p>
<ul>
<li>
<p><a href="https://link.springer.com/article/10.1007/s00253-014-5766-8">Keltjens <em>et al.</em>, 2014 "PQQ-dependent methanol dehydrogenases: rare-earth elements make a difference"</a></p>
</li>
<li>
<p><a href="https://www.cell.com/trends/microbiology/fulltext/S0966-842X(18)30023-4">Chistoserdova and Kalyuzhnaya, 2018 "Current trends in methylotrophy"</a></p>
</li>
</ul>
<p><em>xoxF</em> attracted recently a lot of attention as it encodes one of the few proteins known so far that rely on rare earth elements (REE) as cofactor. This was unexpected as REE were considered to be biologically inert. Have a look at the figure below to get an idea about the effect of REE on cultivating methylotrophs (<strong>Figure 1</strong>).</p>
<p><center><img src="/images/REE_methylos.png" alt="Skovran and Martinez-Gomez, 2015" align="middle" style="width: 500px;"></center></p>
<p><small><b>Figure 1: Increasing the success of isolating/enriching methylotrophs with REEs <a href="http://science.sciencemag.org/content/348/6237/862">(Skovran and Martinez-Gomez, 2015)</a>.</b></small></p>
<p>The agar plate was separated in a way that only the right side was supplemented with REEs. A leaf was used as inoculum, which is common practice for isolating common methylotrophs that settle close to the stomata of leaves, from which small quantities of methanol are regularly released. One can clearly see that REE supplementation leads to a higher number of red-pigmented methylotroph colonies.</p>
<p>Anyway, as you can see I'm getting distracted. What I wanted to do was to create a <em>xoxF</em> reference database for phylogenetic analysis of the <em>xoxF</em> copies present in my isolates. Considering that I could identify multiple copies of <em>xoxF</em> to be present in my isolates I was curious in which of the five known subgroups of <em>xoxF</em> the genes present in my isolates fall.</p>
<h2>Building a XoxF (protein) sequence database</h2>
<p>If you had a look at Keltjens <em>et al.</em>, 2014 you probably saw this nice MxaF/XoxF overview tree (<strong>Figure 2</strong>). I wanted to reproduce a tree like this, adding the sequences from my isolates. </p>
<p><center><img src="/images/keltjens_tree.png" alt="Keltjens et al., 2014" align="middle" style="width: 500px;"></center></p>
<p><small><b>Figure 2: A comprehensive MxaF/XoxF tree <a href="https://link.springer.com/article/10.1007/s00253-014-5766-8">(Keltjens <em>et al.</em>, 2014)</a>.</b></small></p>
<p>Unfortunately, the sequences used for constructing the tree are not provided. The only thing you find in the supplementary material is the uncollapsed tree with all sequences including their accession numbers - annoying, really. Good, whatever, I thought I invest 1 hour and semi-manually download all the sequences from NCBI. <strong>BUT</strong> I was kind of scared after I tried to look up one of the sequences from the tree (<strong>Figure 3</strong>). </p>
<p><center><img src="/images/keltjens_nohit.png" alt="No hit in NCBI, great" align="middle" style="width: 500px;"></center></p>
<p><small><b>Figure 3: When you look up sequences and you are not successful.</b></small></p>
<p>Great, what is going on? Well the paper is already 4 years old and meanwhile the NCBI team started to <a href="https://www.ncbi.nlm.nih.gov/refseq/about/prokaryotes/reannotation/">re-organize</a> its sequence databases to fight the increasing redundancy in their resources (<strong>Figure 4</strong>). That is of course a good thing, but was unfortunate for me.</p>
<p><center><img src="/images/NCBI_YP_WP.png" alt="Tidying up sequence databases" align="middle" style="width: 500px;"></center></p>
<p><small><b>Figure 4: NCBI reorganizes their sequence databases to fight redundancy.</b></small></p>
<p>A lot of the sequences used by Keltjens and colleagues have been reorganized, meaning their accession numbers have changed. In many cases they do not even show up anymore if you search for them naively over all NCBI database resources. Don't you worr! NCBI has nonetheless the brain of an elephant and I made use of that. After assembling a simple .txt file containing all the sequence accessions from the Keltjens tree, I fired up NCBI batch entrez (<strong>Figure 5</strong>) and was able to download the majority of the sequences after selecting the protein database.</p>
<p><center><img src="/images/NCBI_batch_entrez.png" alt="Some bulk downloading" align="middle" style="width: 500px;"></center></p>
<p><small><b>Figure 5: NCBI forgets (almost) nothing, downloading removed records using <a href="https://www.ncbi.nlm.nih.gov/sites/batchentrez">Batch Entrez</a>.</b></small></p>
<p>Not bad, next I did some <em>bash</em> magic to re-format the sequence database to have consistent headers.</p>
<h2>Quick treeing with <em>anvio</em></h2>
<p>I'm very much in love with <a href="http://merenlab.org/software/anvio/"><em>anvio</em></a>, and what is there not like it. It's 100% open source, the developer team is super active (continuously adding features), and it is super well documented. Over the years, <em>anvio</em> can meanwhile not only be used for metagenomics, but for a whole range of applications including and not limited to phylogenomics, pangenomics, structural metagenomics and even "simple" treeing. After assembling my XoxF database, I was now eager to quickly check whether I can reconstruct the tree published by Keltjens <em>et al.</em> using <em>anvio</em>. </p>
<p>Good, before we can use <em>anvio</em> for treeing I used <a href="https://www.drive5.com/muscle/manual/"><em>muscle</em></a> to align the sequences of the databases.</p>
<div class="highlight"><pre><span></span>me@mylaptop:~$ muscle -in PQQ_ADH_Keltjens_renamed.faa -out PQQ_ADH_Keltjens_renamed_aligned.faa
</pre></div>


<p>Next, I used <em>anvio</em> for phylogenetic treeing with <a href="http://www.microbesonline.org/fasttree/"><em>FastTree</em></a>. </p>
<div class="highlight"><pre><span></span><span class="o">(</span>anvio<span class="o">)</span> me@mylaptop:~$ anvi-gen-phylogenomic-tree -f PQQ_ADH_Keltjens_renamed_aligned.faa -o quick_tree.txt
</pre></div>


<p>This should take less than a minute. Please note that I'm using <em>anvio</em> from within a virtual <em>python</em> environment. If you are curious about installing and using <em>anvio</em> have a look <a href="http://merenlab.org/2016/06/26/installation-v2/">here</a>. Now we use the interative interface of <em>anvio</em> to visualize the tree (<strong>Figure 6</strong>).</p>
<div class="highlight"><pre><span></span><span class="o">(</span>anvio<span class="o">)</span> me@mylaptop:~$ anvi-interactive -p tree-profile.db <span class="se">\</span>
                 -t quick_tree.txt <span class="se">\</span>
                 --title <span class="s2">&quot;Quick MxaF/XoxF tree based on the Keltjens et al., 2014 sequence database&quot;</span> <span class="se">\</span>
                 --manual
</pre></div>


<p><center><img src="/images/anvio_tree.png" alt="Easy treeing with anvio" align="middle" style="width: 750px;"></center></p>
<p><small><b>Figure 6: A first MxaF/XoxF tree with <i>anvio</i>.</b></small></p>
<p>And that's it, one hour of work and I have a usable MxaF/XoxF database as well as a first tree that matches the one published in Keltjens <em>et al.</em>, 2014.</p>
<p>Yes, the tree is far from being pretty and needs proper refinement and polishing, but next I can add the XoxF sequences from my isolates to get an idea about their assignment within the different xoxF subclades.</p>
<p>See you</p>
<p>Carl-Eric</p>
            </div>
            <!-- /.entry-content -->
    <hr/>
    <section class="comments" id="comments">
        <h2>Comments</h2>

        <div id="disqus_thread"></div>
        <script type="text/javascript">
            /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
            var disqus_shortname = 'exploring-microbes'; // required: replace example with your forum shortname

                    var disqus_identifier = 'func_genes_db_anvio';
                var disqus_url = 'http://www.exploringmicrobes.de/blog//blog/func_genes_db_anvio/';

            var disqus_config = function () {
                this.language = "en";
            };

            /* * * DON'T EDIT BELOW THIS LINE * * */
            (function () {
                var dsq = document.createElement('script');
                dsq.type = 'text/javascript';
                dsq.async = true;
                dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
                (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
            })();
        </script>
        <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by
            Disqus.</a></noscript>
        <a href="http://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>

    </section>
        </article>
    </section>

        </div>
        <div class="col-sm-3" id="sidebar">
            <aside>
            <!--  -->
            <!--    <div id="aboutme" align="center">
	<h3 style="margin-bottom:0.5px;">Carl-Eric Wegner, PhD</h3><br/>
        <p>
            <center><img width="65%" class="img-thumbnail" src="/images/avatar_2017_b.png" border="0"/></center>
        </p>
    <p>
        I'm a <strong>microbial ecologist</strong> and learning by doing <strong>bioinformatician</strong>. My passion is <strong>microbial carbon cycling</strong>, especially C1-metabolism and complex carbon turnover.
    </p>
</div> -->
            </aside>
        </div>
    </div>
</div>
<footer>
   <div class="well well-lg" id="footer-well">
      <div class="container">
      <div class="row">
           <div class="col-sm-4">
               <h3 style="margin-bottom:0.5px;">Websites I Like</h3>
               </br>
               <ul style="list-style-type:none">
                <div style="margin-bottom: 5.5px; margin-top: 5.5px; font-size: 16px;">
                  <li>
                    <a href="https://www.uni-jena.de" target="_blank">
                        FSU Jena
                    </a>
                  </li>
                <div>
                <div style="margin-bottom: 5.5px; margin-top: 5.5px; font-size: 16px;">
                  <li>
                    <a href="http://www.geomicrobiology.de" target="_blank">
                        KÃ¼sel Lab
                    </a>
                  </li>
                <div>
                <div style="margin-bottom: 5.5px; margin-top: 5.5px; font-size: 16px;">
                  <li>
                    <a href="http://www.aquadiva.uni-jena.de/" target="_blank">
                        AquaDiva
                    </a>
                  </li>
                <div>
                <div style="margin-bottom: 5.5px; margin-top: 5.5px; font-size: 16px;">
                  <li>
                    <a href="https://www.idiv.de/" target="_blank">
                        iDiv
                    </a>
                  </li>
                <div>
              </ul>

             <!-- // <script src="https://apis.google.com/js/platform.js" async defer></script> -->
              <!-- <g:plusone size="standard(bubble)" data-href="http://the-data-show.com"></g:plusone> -->
              <br>
              <h3 style="margin-bottom:0.5px;">Get in touch</h3>
               </br>
               <ul style="list-style-type:none">
                <div style="margin-bottom: 5.5px; margin-top: 5.5px; font-size: 16px;">
                  <li>
                    <a href="http://twitter.com/wegnerce" target="_blank">
		    <i class="fa fa-twitter fa-lg"></i>
                        twitter
                    </a>
                  </li>
                <div>
                <div style="margin-bottom: 5.5px; margin-top: 5.5px; font-size: 16px;">
                  <li>
                    <a href="http://github.com/wegnerce" target="_blank">
		    <i class="fa fa-github fa-lg"></i>
                        github
                    </a>
                  </li>
                <div>
                <div style="margin-bottom: 5.5px; margin-top: 5.5px; font-size: 16px;">
                  <li>
                    <a href="mailto:carl-eric.wegner@uni-jena.de" target="_blank">
		    <i class="fa fa-envelope fa-lg"></i>
                        mail
                    </a>
                  </li>
                <div>
              </ul>
           </div>
           <div class="col-sm-4">
<div id="aboutme" align="center">
	<h3 style="margin-bottom:0.5px;">Carl-Eric Wegner, PhD</h3><br/>
        <p>
            <center><img width="65%" class="img-thumbnail" src="/images/avatar_2017_b.png" border="0"/></center>
        </p>
    <p>
        I'm a <strong>microbial ecologist</strong> and learning by doing <strong>bioinformatician</strong>. My passion is <strong>microbial carbon cycling</strong>, especially C1-metabolism and complex carbon turnover.
    </p>
</div>
           </div>
      </div>
      <div class="row">
         <div class="col-xs-10">&copy; 2019 Carl-Eric Wegner
            &middot; Powered by <a href="http://python.org/" target="_blank">Python</a>,
            <a href="http://docs.getpelican.com/" target="_blank">Pelican</a>,
            a customized <a href="https://github.com/DandyDev/pelican-bootstrap3" target="_blank">pelican-bootstrap3</a>,
            and <a href="http://getbootstrap.com" target="_blank">Bootstrap</a>         </div>
         <div class="col-xs-2"><p class="pull-right"><i class="fa fa-arrow-up"></i> <a href="#">Back to top</a></p></div>
      </div>
   </div>
   </div>
</footer>
<script src="/theme/js/jquery.min.js"></script>

<!-- Include all compiled plugins (below), or include individual files as needed -->
<script src="/theme/js/bootstrap.min.js"></script>

<!-- Enable responsive features in IE8 with Respond.js (https://github.com/scottjehl/Respond) -->
<script src="/theme/js/respond.min.js"></script>

    <!-- Disqus -->
    <script type="text/javascript">
        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
        var disqus_shortname = 'exploring-microbes'; // required: replace example with your forum shortname

        /* * * DON'T EDIT BELOW THIS LINE * * */
        (function () {
            var s = document.createElement('script');
            s.async = true;
            s.type = 'text/javascript';
            s.src = '//' + disqus_shortname + '.disqus.com/count.js';
            (document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
        }());
    </script>
    <!-- End Disqus Code -->


</body>
</html>