Title: Creating a functional gene database, obstacles and a quick tree _anvio_ 
Date: 2019-01-25
Category: Blog.
Tags: bioinformatics, tutorial
Slug: func_genes_db_anvio
Author: Carl-Eric 
Summary: Assembling a quick MxaF/XoxF sequence database and using _anvio_ for a first shot at treeing.

<style>
body {
text-align: justify}
</style>

##What is this about?
 
From work related to the soft coal slags that I started to study a [little while ago](https://www.exploringmicrobes.de/pages/research) I obtained a couple of isolates that turned out to be rather worthwhile for some follow up work. Based on a functional gene-targeting PCR I could see early on that these isolates carry _xoxF_ genes. So far so good, _xoxF_ encodes one type of methanol dehydrogenases (MDH) (yes the isolates are methylotrophs). For a long time it was commonly accepted that menthanol oxidation in methylotrophs is mediated by MxaFI-type MDHs. This perception changed recently, and meanwhile it is hypothesized that XoxF-type MDHs are more widespread in the environment and ecologically more relevant. 

If you are interested in methylotrophy and methanol dehydrogenases (especially MxaFI- versus XoxF-type MDHs) I recommend to have a look at these two publications:

+ [Keltjens _et al._, 2014 "PQQ-dependent methanol dehydrogenases: rare-earth elements make a difference"](https://link.springer.com/article/10.1007/s00253-014-5766-8)

+ [Chistoserdova and Kalyuzhnaya, 2018 "Current trends in methylotrophy"](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(18)30023-4)

_xoxF_ attracted recently a lot of attention as it encodes one of the few proteins known so far that rely on rare earth elements (REE) as cofactor. This was unexpected as REE were considered to be biologically inert. Have a look at the figure below to get an idea about the effect of REE on cultivating methylotrophs (**Figure 1**).

<center><img src="/images/REE_methylos.png" alt="Skovran and Martinez-Gomez, 2015" align="middle" style="width: 500px;"></center>

<small><b>Figure 1: Increasing the success of isolating/enriching methylotrophs with REEs [(Skovran and Martinez-Gomez, 2015)](http://science.sciencemag.org/content/348/6237/862).</b></small>

The agar plate was separated in a way that only the right side was supplemented with REEs. A leaf was used as inoculum, which is common practice for isolating common methylotrophs that settle close to the stomata of leaves, from which small quantities of methanol are regularly released. One can clearly see that REE supplementation leads to a higher number of red-pigmented methylotroph colonies.

Anyway, as you can see I'm getting distracted. What I wanted to do was to create a _xoxF_ reference database for phylogenetic analysis of the _xoxF_ copies present in my isolates. Considering that I could identify multiple copies of _xoxF_ to be present in my isolates I was curious in which of the five known subgroups of _xoxF_ the genes present in my isolates fall.

##Building a XoxF (protein) sequence database

If you had a look at Keltjens _et al._, 2014 you probably saw this nice MxaF/XoxF overview tree (**Figure 2**). I wanted to reproduce a tree like this, adding the sequences from my isolates. 

<center><img src="/images/keltjens_tree.png" alt="Keltjens et al., 2014" align="middle" style="width: 500px;"></center>

<small><b>Figure 2: A comprehensive MxaF/XoxF tree [(Keltjens _et al._, 2014)](https://link.springer.com/article/10.1007/s00253-014-5766-8).</b></small>

Unfortunately, the sequences used for constructing the tree are not provided. The only thing you find in the supplementary material is the uncollapsed tree with all sequences including their accession numbers - annoying, really. Good, whatever, I thought I invest 1 hour and semi-manually download all the sequences from NCBI. **BUT** I was kind of scared after I tried to look up one of the sequences from the tree (**Figure 3**). 

<center><img src="/images/keltjens_nohit.png" alt="No hit in NCBI, great" align="middle" style="width: 500px;"></center>

<small><b>Figure 3: When you look up sequences and you are not successful.</b></small>

Great, what is going on? Well the paper is already 4 years old and meanwhile the NCBI team started to [re-organize](https://www.ncbi.nlm.nih.gov/refseq/about/prokaryotes/reannotation/) its sequence databases to fight the increasing redundancy in their resources (**Figure 4**). That is of course a good thing, but was unfortunate for me.

<center><img src="/images/NCBI_YP_WP.png" alt="Tidying up sequence databases" align="middle" style="width: 500px;"></center>

<small><b>Figure 4: NCBI reorganizes their sequence databases to fight redundancy.</b></small>

A lot of the sequences used by Keltjens and colleagues have been reorganized, meaning their accession numbers have changed. In many cases they do not even show up anymore if you search for them naively over all NCBI database resources. Don't you worr! NCBI has nonetheless the brain of an elephant and I made use of that. After assembling a simple .txt file containing all the sequence accessions from the Keltjens tree, I fired up NCBI batch entrez (**Figure 5**) and was able to download the majority of the sequences after selecting the protein database.

<center><img src="/images/NCBI_batch_entrez.png" alt="Some bulk downloading" align="middle" style="width: 500px;"></center>

<small><b>Figure 5: NCBI forgets (almost) nothing, downloading removed records using [Batch Entrez](https://www.ncbi.nlm.nih.gov/sites/batchentrez).</b></small>

Not bad, next I did some _bash_ magic to re-format the sequence database to have consistent headers.

##Quick treeing with _anvio_

I'm very much in love with [_anvio_](http://merenlab.org/software/anvio/), and what is there not like it. It's 100% open source, the developer team is super active (continuously adding features), and it is super well documented. Over the years, _anvio_ can meanwhile not only be used for metagenomics, but for a whole range of applications including and not limited to phylogenomics, pangenomics, structural metagenomics and even "simple" treeing. After assembling my XoxF database, I was now eager to quickly check whether I can reconstruct the tree published by Keltjens _et al._ using _anvio_. 

Good, before we can use _anvio_ for treeing I used [_muscle_](https://www.drive5.com/muscle/manual/) to align the sequences of the databases.

```bash
me@mylaptop:~$ muscle -in PQQ_ADH_Keltjens_renamed.faa -out PQQ_ADH_Keltjens_renamed_aligned.faa
```

Next, I used _anvio_ for phylogenetic treeing with [_FastTree_](http://www.microbesonline.org/fasttree/). 

```bash
(anvio) me@mylaptop:~$ anvi-gen-phylogenomic-tree -f PQQ_ADH_Keltjens_renamed_aligned.faa -o quick_tree.txt

```

This should take less than a minute. Please note that I'm using _anvio_ from within a virtual _python_ environment. If you are curious about installing and using _anvio_ have a look [here](http://merenlab.org/2016/06/26/installation-v2/). Now we use the interative interface of _anvio_ to visualize the tree (**Figure 6**).

```bash
(anvio) me@mylaptop:~$ anvi-interactive -p tree-profile.db \
                 -t quick_tree.txt \
                 --title "Quick MxaF/XoxF tree based on the Keltjens et al., 2014 sequence database" \
                 --manual
```

<center><img src="/images/anvio_tree.png" alt="Easy treeing with anvio" align="middle" style="width: 750px;"></center>

<small><b>Figure 6: A first MxaF/XoxF tree with <i>anvio</i>.</b></small>


And that's it, one hour of work and I have a usable MxaF/XoxF database as well as a first tree that matches the one published in Keltjens _et al._, 2014.

Yes, the tree is far from being pretty and needs proper refinement and polishing, but next I can add the XoxF sequences from my isolates to get an idea about their assignment within the different xoxF subclades.

See you

Carl-Eric
